### PR TITLE
Enclose sql string by round brackets if there is more than one expressio...

### PR DESCRIPTION
**Pull request by ngdinhtoan**

### CODE REVIEW

**Comment by ngdinhtoan** [comment]
Benchmark new Placeholders():

```
BenchmarkNewPlaceholders	1000000000	         2.08 ns/op
BenchmarkOldPlaceholders	50000000	        36.3 ns/op
```

**Comment by lann** [comment]
This seems at worst harmless, but I'm curious if you have an example where this makes a difference?

File: expr.go [hunk 1]
```diff
@@ -114,6 +111,17 @@ func (c conj) join(sep string) (sql string, args []interface{}, err error) {
 			return "", nil, err
 		}
 		if partSql != "" {
+			switch v := sqlizer.(type) {
+			case Eq:
+				if len(v) > 1 {
+					partSql = fmt.Sprintf("(%s)", partSql)
+				}
```
Comment (lann) [inline expr.go]: Is there a reason this couldn't go above at the end of Eq.toSql?

**Comment by ngdinhtoan** [comment]
I think I just do for safe. But you are right, it's harmless due to AND has higher precedence than OR, and Eq expression only join by AND.

Ref: [Operator Precedence](http://dev.mysql.com/doc/refman/5.0/en/operator-precedence.html)

**Pull request closed without merging**